runtime/compiler/z/codegen/S390PrivateLinkage.cpp

```diff
@@ -3061,15 +3061,14 @@ TR::Register * J9::Z::JNILinkage::buildDirectDispatch(TR::Node * callNode)
      {
      checkException(callNode, methodMetaDataVirtualRegister, methodAddressReg);
      }
-
```

Comment:
It looks like you accidentally removed this blank line.  May I ask you to add it back?